src/client/src/rt-components/platform/adapters/openfin/openFin.ts

```diff
@@ -4,6 +4,15 @@ import { openDesktopWindow } from './window'
 import { fromEventPattern } from 'rxjs'
 import { excelAdapter } from './excel'
 import { CurrencyPairPositionWithPrice } from 'rt-types'
+import { LayoutActions } from '../../../../shell/layouts/layoutActions'
+
+export async function setupWorkspaces(store: any) {
+  if (typeof fin !== 'undefined') {
```

Comment:
You could import the platform here rather than duplicating the openfin check logic
`platform.name=== 'openfin'`

Reply:
Sorry I'm not sure I understand your suggestion. The duplicated part you're talking about, is it with the setupGlobalOpenfinNotifications? because if so, this code is removed in the other PR (related to notification). Are you suggesting to write 
`  const platform = usePlatform()
  if (platform.name === 'openfin')....`
instead of `if (typeof fin !== 'undefined') ` ?

Reply:
I'm gonna merge my current code for now and do another PR next week if needed according to your reply.